tests/utils/execution.hpp

```diff
@@ -147,31 +152,41 @@ void gemm_exec(const std::string& compile_str, size_t batch = 1) {
         result = test_result::skip;
         break;
       }
-
-      auto e_esimd = queue.submit([&](handler& cgh) {
-        cgh.use_kernel_bundle(exeBundle);
-        cgh.parallel_for<Test>(nd_range, [=](nd_item<3> item) KERNEL_MAIN {
-          gpu::xetla::xetla_local_init<SLMSIZE>();
-          gpu::xetla::xetla_nbarrier_init<BARNUM>();
-          KERNEL::run(
-              item,
-              A_ptr,
-              B_ptr,
-              C_ptr,
-              matrix_m,
-              matrix_n,
-              matrix_k,
-              Acc_ptr,
-              Cnt_ptr);
+      for (int iter = 0; iter < iters + warm_up; iter++) {
+        if (iter >= warm_up) {
+          prof.cpu_start();
+        }
+        auto e_esimd = queue.submit([&](handler& cgh) {
+          cgh.use_kernel_bundle(exeBundle);
+          cgh.parallel_for<Test>(nd_range, [=](nd_item<3> item) KERNEL_MAIN {
+            gpu::xetla::xetla_local_init<SLMSIZE>();
+            gpu::xetla::xetla_nbarrier_init<BARNUM>();
+            KERNEL::run(
+                item,
+                A_ptr,
+                B_ptr,
+                C_ptr,
+                matrix_m,
+                matrix_n,
+                matrix_k,
+                Acc_ptr,
+                Cnt_ptr);
+          });
```

Comment:
How did we profile before? and why add it here this time?

Reply:
Not support profile before.

Reply:
In order to get the performance of fp16 xmx